mobile_back_qti/cpp/backend_qti/settings/qti_settings_sd7pg2.pbtxt

```diff
@@ -10,24 +10,36 @@ common_setting {
   }
 }
 
+common_setting {
+  id: "profiling_level"
+  name: "profile level"
+  value {
+    value: "off"
+    name: "profile level"
+  }
+}
+
 benchmark_setting {
   benchmark_id: "image_classification"
-  accelerator: "snpe_dsp"
-  accelerator_desc: "HTP"
   framework: "SNPE"
   custom_setting {
     id: "bg_load"
     value: "true"
   }
-  model_path: "https://github.com/mlcommons/mobile_models/releases/download/v3.0-snpe/mobilenet_edgetpu_224_1.0_htp.dlc"
-  model_checksum: "2e7c4d33b480b5566bdf05e1204b6152"
-  single_stream_expected_latency_ns: 600000
+  delegate_choice: {
+    priority: 1
+    delegate_name: "SNPE_DSP"
+    accelerator_name: "snpe_dsp"
+    accelerator_desc: "HTP"
+    model_path: "https://github.com/mlcommons/mobile_models/releases/download/v3.0-snpe/mobilenet_edgetpu_224_1.0_htp.dlc"
+    model_checksum: "2e7c4d33b480b5566bdf05e1204b6152"
+    single_stream_expected_latency_ns: 600000
```

Comment:
`single_stream_expected_latency_ns` is not a field of `delegate_choice` but `benchmark_setting`, you need to move it one level up.

Files to check:

- `qti_settings_sd7pg2.pbtxt`
- `qti_settings_sd8cxg3.pbtxt`
- `qti_settings_sd8g1.pbtxt`
- `qti_settings_sd8g2.pbtxt`
- `qti_settings_sd8pg1.pbtxt`